Yona/Yona/MeDashBoard/MeDayDetailViewController.swift

```diff
@@ -349,155 +338,160 @@ class MeDayDetailViewController: UIViewController, YonaButtonsTableHeaderViewPro
         return 0.0
     }
     
-    func numberOfSectionsInTableView(_ tableView: UITableView) -> Int {
-        return 2
+    fileprivate func designSpreadCell(_ tableView: UITableView, _ indexPath: IndexPath) -> UITableViewCell {
```

Comment:
Why are you naming these methods ``designXxx``? Are the designing something? It looks more like creating something or finding/getting something.

Reply:
You are right. It's returning cell. I modified methods name to getXxx.